### PR TITLE
chore: Build and Dependency Updates

### PR DESCRIPTION
**Pull request by kingthorin**
Follow-up to zaproxy/zaproxy#6888

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

**Comment by thc202** [comment]
Might require build changes, it's giving an error.

**Comment by kingthorin** [comment]
Ya I got that locally but chalked it up to windows :shrug: guess it's something more.

**Comment by kingthorin** [comment]
Sorted.

File: build.gradle.kts [hunk 1]
```diff
@@ -148,9 +149,9 @@ dependencies {
     testImplementation("org.junit.jupiter:junit-jupiter-params:$jupiterVersion")
     testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:$jupiterVersion")
 
-    testImplementation("io.github.bonigarcia:selenium-jupiter:3.3.0")
+    testImplementation("io.github.bonigarcia:selenium-jupiter:3.4.0")
```
Comment (kingthorin) [inline build.gradle.kts:152]: There is a 5.x but it breaks a bunch of tests and I didn’t have time to dig into it.

**Comment by thc202** [comment]
The build is now saying that there are deprecations, I'm fine merging as is since this is included in the weekly and better use the same version.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by ricekot — APPROVED** [review]

**Pull request merged**